### PR TITLE
Create social-science-history.csl

### PR DESCRIPTION
**Pull request by danielscottsmith**
A modified Chicago 16th in compliance with Social Science History guidelines printed here: https://ssha.org/files/SSHStyleGuide.pdf

### CODE REVIEW

File: social-science-history.csl [hunk 1]
```diff
@@ -0,0 +1,619 @@
...
+        <group delimiter=", " prefix=", in ">
+          <names variable="editor translator" delimiter=", ">
+            <name and="text" delimiter=", "/>
+            <label form="short" prefix=" (" suffix=")"/>
+          </names>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="editor">
```
Comment (POBrien333) [inline social-science-history.csl:56]: delete this unused macro

File: social-science-history.csl [hunk 2]
```diff
@@ -0,0 +1,619 @@
...
+    <names variable="editor">
+      <name name-as-sort-order="first" and="text" sort-separator=", " delimiter=", " delimiter-precedes-last="always"/>
+      <label form="short" prefix=", "/>
+    </names>
+  </macro>
+  <macro name="translator">
```
Comment (POBrien333) [inline social-science-history.csl:62]: delete this unused macro

**Comment by danielscottsmith** [comment]
From my inspection, the CSL is compliant with most of the guidelines specified here:https://ssha.org/files/SSHStyleGuide.pdf

There is one exception in the reference list I can identify.

Expected behavior:
Ross, Dorothy (2003) “Changing Contours of The Social Science Disciplines,” in Theodore M Porter and Dorothy Ross (eds.) The Modern Social Sciences. Vol. 7, The Cambridge History of Science. Cambridge: Cambridge University Press: 1–25.

Observed behavior:
Ross, Dorothy (2003) “Changing Contours of The Social Science Disciplines,” in Theodore M Porter and Dorothy Ross (eds.) The Modern Social Sciences. Cambridge: Cambridge University Press: 7:1–25.

_When page numbers are included in the citation, then the label "Vol." is missing and the volume number is put at the end as a prefix to the page numbers. The citation appears as expected when page numbers are not included._

**Comment by adam3smith** [comment]
Sorry for the delay & thanks for the detailed description of the remaining issue -- I fixed that before merging.

**Comment by danielscottsmith** [comment]
brilliant; you're the best! thanks for all the selfless work~

**Pull request merged**